.ci/appveyor.yml

```diff
@@ -3,16 +3,16 @@ environment:
     # Pre-installed Python versions, which Appveyor may upgrade to
     # a later point release.
     # See: http://www.appveyor.com/docs/installed-software#python
-    - PYTHON_VERSION: "3.5" # currently 3.5.1
+    - PYTHON_VERSION: "3.6" # currently 3.5.1
```

Comment:
@renefritze Do you want to update the `3.5.1` in the comment?

Reply:
Good catch! The appveyor builds will probably stay broken (/disabled) until I get to the conda packages for windows though.